html/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/style.css" />
+
+    <link rel='stylesheet'
+    href='https://fonts.googleapis.com/css?family=Poppins' >
+
+      <link rel="stylesheet" 
+      href="path/to/font-awesome/css/font-awesome.min.css">
+
+      <link  rel="stylesheet" 
+      href="http://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css">
+
+    <title>James Port-Folio</title>
+  </head>
+  <body>
+    <header class="header-title">
+        <div class="navbar">
+          <a><p>James</p></a>
+          <div class="navbar-list">
+            <a href="#" > Porto-folio </a>
+            <a href="#" > About </a>
+            <a href="#" > Contact </a>
+            <a href="#" class="icon"><i class="fa fa-bars"></i></a>
+          </div>
```

Comment:
- [ ] Kindly use nav tag instead of div to improve SEO for your website
- [ ] kindly I would recommend you to stick to [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design as much as possible to match it so that your project looks professional .you can do so by using the same margins, font colors, font style, padding, flexbox, background-image for the mobile version, etc.